package.json

```diff
@@ -17,8 +17,9 @@
     "express": "^4.17.1",
     "express-cache-controller": "^1.1.0",
     "limiter": "^1.1.5",
-    "mariadb": "^2.4.0",
-    "memory-cache": "^0.2.0"
+    "mariadb": "^2.4.1",
+    "memory-cache": "^0.2.0",
+    "mysql": "^2.18.1"
```

Comment:
this shouldn't be needed since we are already using the mariadb connector.

Reply:
Is this just referring to the mysql? I have removed that now